streamx-plugin/streamx-flink-packer/src/main/scala/com/streamxhub/streamx/flink/packer/docker/DockerTool.scala

```diff
@@ -44,7 +44,7 @@ object DockerTool extends Logger {
     }
     // generate dockerfile
     val dockerfile = dockerFileTemplate.writeDockerfile(projectBaseDir)
-    val tagName = compileTag(expectImageTag, authConf.registerAddress)
+    val tagName = compileTag(expectImageTag, authConf.registerAddress).toLowerCase
```

Comment:
Maybe it will be better
https://github.com/streamxhub/streamx/blob/6aa8cbe47ba8728267b73640453ffb2f32f8a55e/streamx-plugin/streamx-flink-packer/src/main/scala/com/streamxhub/streamx/flink/packer/docker/DockerTool.scala#L114
```scala
  def formatTag(tag: String, registerAddress: String): String = {
    if (registerAddress.nonEmpty && !tag.startsWith(registerAddress)) {
      s"$registerAddress${if (tag.contains("/")) "/" else "/library/"}$tag".toLowerCase
    } else tag.toLowerCase
  }
```